templates/taxi/car_list.html

```diff
@@ -1,17 +1,21 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car List</h1>
+  <a href="{% url 'taxi:car-create' %}">Create new Car</a>
```

Comment:
Code Style: Please make sure to use 2 whitespaces for indentation in your `.html` files. Currently, your indentation seems to be more than 2 whitespaces.